### PR TITLE
Tweak changelog entry placeholder

### PR DESCRIPTION
**Pull request by timja**
(head branch, delete on merge)

I'm constantly removing `Entry 1:` from entries, otherwise they end up in the changelog by accident...
This should hopefully work better?

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/8532"></a>

### CODE REVIEW

**Review by daniel-beck — APPROVED** [review]
What's the likelihood of the `<>` remaining? 😁

**Review by NotMyFault — APPROVED** [review]

**Comment by timja** [comment]
> What's the likelihood of the `<>` remaining? 😁

Yeah i'm a little worried about that, if I see it happening I'll remove it in a second iteration.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -28,12 +28,15 @@ For refactoring and code cleanup changes, exercise the code before and after the
 
 ### Proposed changelog entries
 
-- Entry 1: Issue, human-readable text
-- […]
+- \<replace with Issue ID if it exists or remove\>, human-readable text
```
Comment (Vlatombe) [inline .github/PULL_REQUEST_TEMPLATE.md:31]: That pattern got me confused to be honest. I think giving an example would work better.

Comment (daniel-beck) [reply .github/PULL_REQUEST_TEMPLATE.md:31]: Noting previously negative feedback I got in https://github.com/jenkinsci/jenkins/pull/7236#discussion_r996221993

**Pull request merged**